embasi/embedding.py

```diff
@@ -906,3 +919,173 @@ def run(self):
         from scalapack4py.npscal.blacs_ctxt_management import CTXT_Register, DESCR_Register
         CTXT_Register.clear_register()
         DESCR_Register.clear_register()
+
+
+class FrozenDensityEmbedding(EmbeddingBase):
+    """Implementation of Projeciton-Based Embedding with ASI communication
+    A class controlling the interaction between two subsystems calculated at
+    two levels of theory (low-level and high-level) with the
+    Projection-Based Embedding (PbE) scheme of Manby et al.[1].
+    Parameters
+    ----------
+    atoms: ASE Atoms
```

Comment:
The documentation is automatically generated using the Numpydocs template, so having the right input/output arguments in place will be good to avoid having to write explicit documentation for the class in the doxygen.